extensions/amp-form/0.1/amp-form.js

```diff
@@ -1620,6 +1623,35 @@ export class AmpFormService {
       this.installSubmissionHandlers_(doc.querySelectorAll('form'));
     });
   }
+
+  /**
+   * Listen for Ctrl/Cmd + Enter in textarea elements
+   * to trigger form submission when relevant.
+   * @param {!Document|!ShadowRoot} doc
+   */
+  installFormSubmissionShortcutForTextarea_(doc) {
+    doc.addEventListener('keydown', e => {
+      if (
+        e.key != Keys.ENTER ||
+        !(e.ctrlKey || e.metaKey) ||
+        e.target.tagName !== 'TEXTAREA'
+      ) {
+        return;
+      }
+      const formId = e.target.getAttribute('form');
```

Comment:
Can we use the `form` property here? https://developer.mozilla.org/en-US/docs/Web/API/HTMLTextAreaElement